packages/dbml-homepage/docs/.vuepress/config.js

```diff
@@ -18,5 +18,14 @@ module.exports = {
         ''
       ],
     }
-  }
+  },
+  ga: `${require('./secret').ga}`,
```

Comment:
I think `ga` only set in vuepress plugin config

Reply:
My bad, i should have removed it first.